src/dash.ts

```diff
@@ -16,6 +17,10 @@ export class Dash {
             uri += '&keys=' + keys;
         }
 
+        if (this.usingZeal()) {
```

Comment:
Hi @szhongren, thanks for making this PR!

is it possible if we just detect the operating system? 

e.g
```
var isWindows = os.platform() === 'win32';

if (isWindows)  {
  return 'zeal` ....
}
```
Ref: https://nodejs.org/api/os.html